src/main/java/ua/softserveinc/tc/service/impl/BookingServiceImpl.java

```diff
@@ -194,7 +193,7 @@ private boolean isAvailabilePlacesInTheRoom(List<BookingDto> listDTO) {
     @Override
     public List<BookingDto> persistBookingsFromDtoAndSetId(List<BookingDto> listDTO) {
         if(listDTO == null)
-            throw new RuntimeException("listDTO is null");
+            throw new NullPointerException("listDTO is null");
```

Comment:
Ok. But in this case we are going to have possible problem without checking for null.